src/scss/base/_grid.scss

```diff
@@ -225,8 +230,8 @@
     // Generates gutter settings for each defined breakpoint for better gutter control in responsive designs
     // Only generated if $responsive-gutters == true
 
-    @if $responsive-gutters {
-        @each $name, $value in $breakpoints {
+    @if v.$responsive-gutters {
+        @each $name, $value in v.$fozzie-breakpoints {
```

Comment:
The `v.$fozzie-breakpoints` references in this file are breaking for me when referencing this module in another project.

I think they need to use `$fozzie-breakpoints` from `'../tools/helpers/breakpoints'`, right?

Reply:
https://github.com/justeat/fozzie/pull/388

Reply:
Thanks!